doc.go

```diff
@@ -46,11 +52,17 @@ The connection string (DSN) can contain both connection parameters (described be
 The following connection parameters are supported:
 
 	* account <string>: Specifies the name of your Snowflake account, where string is the name
-		assigned to your account by Snowflake. In the URL you received from
-		Snowflake, your account name is the first segment in the domain (e.g.
-		abc123 in https://abc123.snowflakecomputing.com). This parameter is
-		optional if your account is specified after the @ character. If you
-		are not on us-west-2 region or AWS deployment, then append the region
+		assigned to your account by Snowflake.
+
+		For newer accounts, and for older accounts converted to use the Organizations feature,
```

Comment:
"Newer" vs. "older" is not a distinction that's going to be meaningful a year from now.
We should just point to https://docs.snowflake.com/en/user-guide/admin-account-identifier.html and leave it to that page to explain the different formats for specifying the account identifier.

Reply:
Updated.